include/z64actor.h

```diff
@@ -296,33 +296,47 @@ typedef enum {
 #define ACTORCTX_FLAG_6 (1 << 6)
 #define ACTORCTX_FLAG_7 (1 << 7)
 
-
-typedef struct {
+// A Lock On entry is a set of 4 triangles which appear around an actor when the player Z-Targets it
+typedef struct TargetLockOnEntry {
     /* 0x00 */ Vec3f pos;
-    /* 0x0C */ f32 unkC;
+    /* 0x0C */ f32 radius; // distance towards the center of the locked on
     /* 0x10 */ Color_RGBA8 color;
-} TargetContextEntry; // size = 0x14
+} TargetLockOnEntry; // size = 0x14
 
 typedef struct TargetContext {
-    /* 0x00 */ Vec3f unk0;
+    /* 0x00 */ Vec3f fairyHintPos; // Used by Tatl to indicate a targetable actor or general hint
     /* 0x0C */ Vec3f targetCenterPos;
     /* 0x18 */ Color_RGBAf fairyInner;
     /* 0x28 */ Color_RGBAf fairyOuter;
-    /* 0x38 */ Actor* arrowPointedActor;
+    /* 0x38 */ Actor* targetableOption;
     /* 0x3C */ Actor* targetedActor;
```

Comment:
Should `targetedActor` be changed to `lockOnActor` in both `TargetContext` and `Player`?